From: Dmitry Shachnev <mitya57@debian.org>
Date: Sat, 8 Feb 2020 13:12:45 +0300
Subject: Unset body max-width option

Sphinx 1.7+ sets it to 800px by default, which does not play nice with
our right sidebar.
---
 doc/conf.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/doc/conf.py b/doc/conf.py
index 358414f..cccfd95 100644
--- a/doc/conf.py
+++ b/doc/conf.py
@@ -199,6 +199,7 @@ html_theme_options = {
     'relbarbgcolor': '#fff',
     'relbartextcolor': '#20435c',
     'relbarlinkcolor': '#355f7c',
+    'body_max_width': None,
 }
 
 # the css mostly overrides this:
